md/index.md

```diff
@@ -10,8 +10,12 @@ Click on the **links** below to find useful information quickly, or [browse the
 <div class="panel-body">
 <div class="menu-block-wrapper">
 
-* [Release notes ${varVersion}](release-notes.md)    <!--{li:.first .leaf}-->
-
+* [Release notes ${varVersion}](release-notes.md)<!--{li:.first .leaf}-->
+* [Collaborative development with Git](workspaces-and-repositories.md#git)<!--{li:.first .leaf}-->
+* [Business data access control (BDN AC)](bdm-access-control.md)<!--{li:.first .leaf}-->
+* [Save a form](release-notes.md#save-form)<!--{li:.first .leaf}-->
+* [New case overview form](release-notes.md#new-case-overview)<!--{li:.first .leaf}-->
```

Comment:
Please be aware that when you add a link to an anchor like **new-case-overview**, you should do it in the same PR as the one that introduces the anchor, in order for the link to be able to be tested.
Otherwise, it is considered a dead link.
If you need help to create PRs on more than one file, do not doubt in asking one of us. We'll be proud to show off !